packages/components/src/ui/radix-dropdown/index.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+import * as DropdownMenuPrimitive from '@radix-ui/react-dropdown-menu';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+import { SVG, Circle } from '@wordpress/primitives';
+import { check, lineSolid } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import Icon from '../../icon';
+import * as DropdownMenuStyled from './styles';
+import type {
+	DropdownMenuProps,
+	DropdownSubMenuProps,
+	DropdownItemProps,
+} from './types';
+
+// Observations / Questions:
+// - is it enough on the larger components to have only one forwarded ref?
+//   If we have only one, should it be to the "root", the "trigger", or the "content"?
```

Comment:
For now, let's start with no ref at all

Reply:
I removed ref forwarding for most components, although it seems like refs are required in some cases, especially when using the `asChild` prop

Reply:
All `asChild` props have been removed, apart from the trigger component